examples/with-react-hook-form/components/Login.js

```diff
@@ -0,0 +1,54 @@
+import { useForm } from 'react-hook-form'
+
+const Login = () => {
+  const { register, errors, handleSubmit } = useForm()
+  const login = handleSubmit(({ username, password, remember }) => {
+    // Handle login logic with username, password and remember form data
```

Comment:
Let's pop an alert with the data. Something interactive!

Reply:
I made changes, so now when form validation is ok then user name shows up.